samples/ingestion/ingestion-client/Connector/TransientFailureException.cs

```diff
@@ -0,0 +1,30 @@
+﻿// <copyright file="TransientFailureException.cs" company="Microsoft Corporation">
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE.md file in the project root for full license information.
+// </copyright>
+
+namespace Connector
+{
+    using System;
+
+    /// <summary>
+    /// Exception that is being thrown when azure functions should trigger a retry (up to a max. retry limit)
+    /// after a certain number of minutes.
```

Comment:
I guess it's seconds rather than minutes for you?

Reply:
no, it is actually minutes (either 1,2,4... and so on) :)